src/restaurant/repository/restaurant.repository.ts

```diff
@@ -0,0 +1,13 @@
+import { Restaurant } from './../entity/restaurant.entity';
+import { EntityRepository, Repository } from 'typeorm';
+
+@EntityRepository(Restaurant)
+export class RestaurantRepository extends Repository<Restaurant> {
+  getRestaurantById(id: number) {
+    return this.findOne({ where: { id } });
+  }
+
+  saveWithRestaurant(restaurant: Restaurant) {
+    return this.save(restaurant);
```

Comment:
엔티티를 미리 생성해서 기본값을 채워줍니다. (있는경우)
```suggestion
    return this.save(this.create(restaurant));
```

Reply:
해당 함수의 경우 Input값을 받아와서 엔티티 생성후 save하는 Insert개념이 아니라
Restaurant 타입을 받아와서 그대로 저장하는 update의 개념으로 만든거여서 굳이 create가 필요없지 않나요?

Reply:
이 부분 그냥 save 해도 되는데 제가 괜히 만든것 같아요 수정해놓겠습니다~